src/debugger/config.clj

```diff
@@ -2,7 +2,7 @@
   (:require [debugger.time :as t]))
 
 (declare ^:dynamic *locals*)
-(def ^:dynamic *break-outside-repl* false)
+(def ^:dynamic *break-outside-repl* true)
```

Comment:
I think, it will require a minor bump at least, could you mention this change in readme?